query/sources/common/src/main/java/org/codice/ddf/admin/sources/utils/RequestUtils.java

```diff
@@ -164,29 +189,85 @@ public ReportImpl endpointIsReachable(UrlField urlField) {
         return report;
     }
 
-    private WebClient generateClient(String url, CredentialsField creds,
-            Map<String, String> queryParams) {
-        SecureCxfClientFactory<WebClient> clientFactory = createFactory(url, creds);
+    public class WebClientBuilder {
```

Comment:
Just read https://stackoverflow.com/questions/43998428/why-to-favor-static-member-classes-over-nonstatic-joshua-bloch-item-22 . Going to change it to static if possible.